packages/common/src/search/types/types.ts

```diff
@@ -82,7 +82,7 @@ export interface IApiDefinition {
   // - for "arcgis-hub", the /v3/search will be added
   url: string;
   // We can add types as we add support for more
-  type: "arcgis" | "arcgis-hub";
+  type: "arcgis" | "arcgis-hub" | "discussions";
```

Comment:
Check with @sonofflynn89 re: how we are going to pass the api type into the gallery - it's changed since the early implementation and I'm not sure how it's done.

Reply:
@dbouwman I already went over the PR with Josh in a call and gave him some action items. There are some serious considerations we'll need to make before this stuff is gallery ready.

This API type is exactly the type of thing I was hoping you could give Josh more guidance on. Since the discussions API is technically a Hub API, I was initially wary about adding this additional type. However I am ok leaving this as is IF we think that we will ever need to get discussion channels via the OGC api instead of accessing the discussions API directly. What do you think?